frontend/src/modules/widget/components/cube/_query_builder/DimensionSelect.vue

```diff
@@ -64,6 +64,7 @@ export default {
           'Activities.platform',
           'Activities.type',
           'Activities.date',
+          'Activities.channel',
```

Comment:
You will also need to update `en.js`
In
```
widget: {
  cubejs: {
      Activities: {
        count: '[Activities] Count',
        type: '[Activities] Type',
        platform: '[Activities] Platform',
        date: '[Activities] Date'
      },
    }
}
```
Just add `channel: '[Activities] Channel'` within the object